### PR TITLE
Radar in Meters

### PR DESCRIPTION
**Pull request by TimGoll**
The builtin radar now shows meters instead of hammer units

### CODE REVIEW

File: lua/terrortown/entities/items/item_ttt_radar/cl_init.lua [hunk 1]
```diff
@@ -168,7 +170,7 @@ function RADAR:Draw(client)
 		end
 	end
 
-	-- Corpse calls
+	-- vorpse calls
```
Comment (saibotk) [inline lua/terrortown/entities/items/item_ttt_radar/cl_init.lua:173]:
```suggestion
	-- corpse calls
```

File: lua/terrortown/lang/en.lua [hunk 2]
```diff
@@ -2174,3 +2174,6 @@ L.header_weapon_settings = "Weapon Settings"
 
 -- 2024-01-24
 L.grenade_fuse = "FUSE"
+
+-- 2024-02-04
+L.distance_meter = "{distance}m"
```
Comment (saibotk) [inline lua/terrortown/lang/en.lua:2179]: I think "m" is a pretty universal standard globally and does not need to be translated

**Review by saibotk — CHANGES_REQUESTED** [review]

File: lua/terrortown/lang/en.lua (hunk 2 above)
Comment (TimGoll) [reply lua/terrortown/lang/en.lua:2179]: I'm not sure it is m in chinese etc

Comment (saibotk) [reply lua/terrortown/lang/en.lua:2179]: It is the official SI unit symbol there is nothing to translate

**Review by saibotk — APPROVED** [review]

**Comment by EntranceJew** [comment]
is this consistent with the DNA scanner's display?

**Comment by TimGoll** [comment]
> is this consistent with the DNA scanner's display?

good point, no. Will look at this in another PR

**Review by Histalek — APPROVED** [review]

**Pull request merged**